### PR TITLE
added multicodec-packed beginning

### PR DESCRIPTION
**Pull request by jbenet**
The table is still missing.

### CODE REVIEW

**Comment by jbenet** [comment]
RFC @whyrusleeping @diasdavid @RichardLitt @kevina

**Comment by RichardLitt** [comment]
LGTM!

**Comment by diasdavid** [comment]
LGTM as well :) Thank you!

File: multicodec-packed.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+This is the standard multicodec-packed protocol table.
+
+#### WARNING: WIP. this table is not ready for wide use.
+
+TODO: see if IANA has a ready-made table for us to use here. Even just a listing of the most popular formats would be good enough.
+
+```sh
+code  codec
+0x00  raw binary data
+0x40  multicodec
```
Comment (kevina) [inline multicodec-packed.md]: This is self referencing.  Is the idea to use this when a packed-multicodec does not describe the data?

**Comment by kevina** [comment]
I have one question.  Other than that it LGTM.

File: multicodec-packed.md (hunk 1 above)
Comment (jbenet) [inline multicodec-packed.md]: this meant regular multicodec, not mutlicodec-packed

**Comment by jbenet** [comment]
Okay, i'm going to merge this. we can work on the table next.

**Pull request merged**